client/src/modules/list/index.jsx

```diff
@@ -128,6 +123,13 @@ class List extends Component {
               <InputBar />
             </ItemsContainer>
             <ItemsContainer archived items={orderedItems} />
+            <button
+              className="link-button"
+              onClick={this.handleDialogContext(DialogContext.ARCHIVE)}
+              type="button"
+            >
+              {`Archive the ${name} list`}
```

Comment:
surround `name` with `"` signs - this should be done everywhere we use list name, otherwise we get sometimes strange sentences